collection-spec/collection-spec.md

```diff
@@ -1,13 +1,13 @@
 # STAC Collection Specification
 
 The STAC Collection Specification defines a set of common fields to describe a group of Items that share properties and metadata. The 
-Collection Specification extends the STAC [Catalog Specification](../catalog-spec/catalog-spec.md) with additional fields to 
-describe the whole dataset and the included set of Items. It shares the same fields and therefore every Collection is also a valid Catalog. Collections can have both parent Catalogs and Collections and child Items, Catalogs and Collections. 
+Collection Specification shares some fields with the STAC [Catalog Specification](../catalog-spec/catalog-spec.md) with additional fields to 
```

Comment:
Should we say it shares all files with different semantics for stac_extensions and type?